### PR TITLE
Fix PHPUnit warnings on NODB runs

### PR DESCRIPTION
**Pull request by rullzer**
Lots of getMock fixes.
Cleans up the unit test output significantly and makes spotting new warnings a lot easier!

CC: @MorrisJobke @nickvergessen @LukasReschke @icewind1991

### CODE REVIEW

File: apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php [hunk 1]
```diff
@@ -195,7 +195,7 @@ public function testValidateUsedCode() {
 			->with($user)
 			->will($this->returnValue($codes));
 		$this->hasher->expects($this->never())
-			->method('verifiy');
+			->method('verify');
```
Comment (MorrisJobke) [inline apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php]: 🙈

Comment (ChristophWurst) [inline apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php]: :hear_no_evil:

File: tests/lib/AppFramework/Db/MapperTestUtility.php [hunk 2]
```diff
@@ -49,7 +49,7 @@ protected function setUp(){
 			->disableOriginalConstructor()
 			->getMock();
 
-		$this->query = $this->getMock('\PDOStatement');
+		$this->query = $this->createMock('\PDOStatement');
```
Comment (ChristophWurst) [inline tests/lib/AppFramework/Db/MapperTestUtility.php]: why didn't you use the `::class` property here?

**Comment by ChristophWurst** [comment]
:+1: looks good

File: tests/lib/AppFramework/Db/MapperTestUtility.php (hunk 2 above)
Comment (rullzer) [inline tests/lib/AppFramework/Db/MapperTestUtility.php]: ah yeah I could now probabaly Just forgot.

**Comment by LukasReschke** [comment]
LGTM

File: apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php (hunk 1 above)
Comment (nickvergessen) [inline apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php]: shouldn't this test have failed because of "unmockable method"?

Comment (rullzer) [inline apps/twofactor_backupcodes/tests/Unit/Service/BackupCodeStorageTest.php]: Phpunit warns about it.

**Pull request merged**